src/chat/components/ViewUnRead.tsx

```diff
@@ -22,8 +23,11 @@ const ViewUnRead: React.FC<ViewUnReadProps> = ({
   customTextStyle,
   unReadSentMessage = 'Sent',
   unReadSeenMessage = 'Seen',
+  viewUnReadCustom,
 }) => {
-  return (
+  return viewUnReadCustom ? (
+    viewUnReadCustom()
```

Comment:
```suggestion
    viewUnReadCustom(userUnreadMessage)
```
Link to the above comment

Reply:
ok I updated